index.html

```diff
@@ -59,227 +60,218 @@ <h1>I'm Abdi</h1>
         <li><a href="#" class ="link"><img src="./assets/images/M.svg"></a></li>
      </ul>
  </div>
-    
+
+
 </section>
     </div>
+       
+            <!-- ******* Work Section  *******-->  
     <section class="container">
         <div class="works-Section">
             <h2 class="recentWorks_header">My Recent Works</h2>
+            <img src="./assets/images/besideWorkLine.png" id="besiderecentWorks">
+            
+            <!--Card1-->
             <div class="cards card1">
-                <img src="./assets/images/multi-Post-Story.png">
-                <h3 class="card-title">Multi-Post Stories</h3>
-                <div class="card1_description">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s,
-                         when an unknown printer took a standard dummy text.</p>
-                </div>
-                
-                <div class="cards_buttons">
-                    <ul>
+          <div class="card1_img">    <img src="./assets/images/multi-Post-Story.png" class="multi_post"></div>  
+                <div id="card1_container">
+                 <h3 class="card-title">Multi-Post Stories</h3>
+                  <div class="card1_description">
+                     <p>A daily selection of privately personalized reads;
+                         no accounts or sign-ups required. has been the industry's standard dummy
+                         xt ever since the 1500s,
+                         when an unknown printer took a standard dummy text.
+                     </p>
+                  </div>
+                   <div class="cards_buttons">
+                     <ul>
                         <li> <a href="#">  <button type="button" id="button_link">css</button></a></li>
                         <li> <a href="#">  <button type="button" id="button_link">html</button></a></li>
                         <li>  <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
                         </li>
                         <li> <a href="#">  <button type="button" id="button_link">Ruby</button></a></li>
-                  
-                  
-                 
-                  
-                   
-               </ul>
-
-                </div>
-                <div class="seeProject">
+                    </ul>
+                  </div>
+                   <div class="seeProject">
```

Comment:
[OPTIONAL]For all your buttons, it would be really great if you could implement hover. This is part of the requirements for previous milestones. This feature gives your page semantic relevance and makes user experience better.
Remember to use the design given on your figma template.
<img width="380" alt="Screenshot 2022-09-30 at 23 45 07" src="https://user-images.githubusercontent.com/48546992/193353617-b307b27b-88e6-4ea9-9515-de5ab48dcdff.png">